static/app/views/settings/project/projectKeys/details/index.tsx

```diff
@@ -37,7 +38,9 @@ export default class ProjectKeyDetails extends AsyncView<Props, State> {
   handleRemove = () => {
     const {organization} = this.props;
     const {projectId} = this.props.params;
-    browserHistory.push(`/${organization.slug}/${projectId}/settings/keys/`);
+    browserHistory.push(
+      normalizeUrl(`/settings/${organization.slug}/projects/${projectId}/keys/`)
+    );
```

Comment:
Another updated redirect. I also took the liberty of updating the route to not use a sentry9 path.